src/fetching/RelayConnectionFetcher.js

```diff
@@ -72,22 +76,23 @@ export default class RelayConnection<T, Data> implements Fetcher<Data[]> {
 		};
 
 		while (retries < this.maxRetries) {
-			const res = await fetch(url, options);
-			const data = await res.json();
-
-			if (data.errors && Array.isArray(data.errors)) {
+			try {
+				const res = await fetch(url, options);
+				if (retrieCodes.includes(res.statusCode)) {
```

Comment:
I think you're looking for `status`. See [here](https://developer.mozilla.org/en-US/docs/Web/API/Response/status)